dist/index.html

```diff
@@ -0,0 +1 @@
+<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Webpack Set-Up</title><script src="https://kit.fontawesome.com/64d25ab310.js" crossorigin="anonymous"></script><script defer="defer" src="./main.js"></script></head><body><section id="main-container"><div id="main-c-head"><h3>Today's To Do</h3><i class="fa-solid fa-arrows-rotate"></i></div><input placeholder="Add to your list" id="to-do-input"><article id="to-do-list"></article><p id="clear-completed" class="clear-completed">Clear all completed</p></section></body></html>
```

Comment:
- [ ] Kindly put the document in order, line by line. It is easier to review and use the inline comments in the corresponding place. 📑 👀